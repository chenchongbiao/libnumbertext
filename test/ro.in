RON 1
RON 1,1
RON 1,10
