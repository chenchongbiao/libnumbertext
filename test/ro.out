un leu românesc
un leu românesc zece bani
un leu românesc zece bani
